etc/amazon-corretto-crypto-provider.security

```diff
@@ -46,7 +46,6 @@ security.provider.17=JdkSASL
 security.provider.18=sun.security.mscapi.SunMSCAPI
 security.provider.19=SunMSCAPI
 security.provider.20=apple.security.AppleProvider
-security.provider.21=Apple
```

Comment:
Was this change intended?

Reply:
yes, there's an explanation in the last paragraph of the Notes/commit message.

Reply:
updated explanation:

> Now that this runs on Mac the Apple JCE providers in our pre-baked properties
> files are actually used (they're ignored on Linux, where they're unavailable).
> This resulted in the same provider being installed twice, causing
> SecurityPropertiesTester failures. We resolve this issue by removing the
> class-specified entry in favor of the name-specified entry.  Because the
> [AppleProvider][3] is [not exported][4] from its module in JDK 11, we cannot
> use the fully qualified class specification (see comment in
> `etc/amazon-corretto-crypto-provider.security`) on JDK 9+. By using only the
> name specification, there may be some issues on JDK8 on Mac when using this
> providers file, but the impact should be minor -- no regressions (because we
> didn't support MacOS before). Users are encouraged to upgrade to more moder
> JDKs or manually modify/override the supplied properties file.

[3]: https://github.com/corretto/corretto-11/blob/develop/src/java.base/macosx/classes/apple/security/AppleProvider.java
[4]: https://github.com/corretto/corretto-11/blob/develop/src/java.base/macosx/classes/module-info.java.extra